libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -296,6 +284,17 @@ public boolean areFamilyNameAndDobCorrelatedForCoiCheck(List<VerifiableCredentia
         return true;
     }
 
+    public List<VerifiableCredential> getSuccessfulVcs(List<VerifiableCredential> vcs)
```

Comment:
Not very important, but does this need to be public?

Reply:
oops! Nice spot, no it doesn't I just forgot to put it back to private after refactoring